bib/cmd/bootc-image-builder/image.go

```diff
@@ -514,6 +514,7 @@ func manifestForISO(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest, erro
 			BIOS:       true,
 			UEFIVendor: c.SourceInfo.UEFIVendor,
 		}
+		img.ISOBoot = manifest.Grub2ISOBoot
```

Comment:
The original code in images (c.f. https://github.com/osbuild/images/pull/1289/files#diff-5c50c723b0bc0e23febb416e111b78907d5463429d05202682fa7e0ef0588c71R134) had syslinux iso for x86_64, I think moving to grub2ISOBoot instead is fine but just want to be explicitly call it out to double check with the rest of the team (cc @achilleas-k and @bcl )